src/python/py/models/quantized_model.py

```diff
@@ -52,51 +52,86 @@ def __init__(self):
         self.weight = None
         self.bias = None
 
+    @property
+    def default(self):
+        return self
 
-class QuantizedAttention:
+
+class QuantizedLoRAModule(QuantizedTensorModule):
     def __init__(self, bits, group_size):
-        self.q_proj = QuantizedTensorModule(bits, group_size)
-        self.k_proj = QuantizedTensorModule(bits, group_size)
-        self.v_proj = QuantizedTensorModule(bits, group_size)
-        self.o_proj = QuantizedTensorModule(bits, group_size)
+        super().__init__(bits, group_size)
+        self.lora_A = TensorModule()
+        self.lora_B = TensorModule()
+        self.scaling = {"default": 1}
```

Comment:
If `scaling` will be calculated from the config since it cannot be determined from the `.safetensors` files, can this be removed?